language-plutus-core/test/types/example.plc.golden

```diff
@@ -1 +1 @@
-(all a (type) (fun (fix x (all r (type) (fun r (fun (fun a (fun x r)) r)))) a))
\ No newline at end of file
+Unexpected 'fix' at 3:24
```

Comment:
These probably weren't meant to be error tests :)

Reply:
Whoops :)
Well, that's all very raw at the moment.